src/storage.ts

```diff
@@ -15,9 +15,14 @@
  */
 
 import { openDB, DBSchema, IDBPDatabase } from 'idb';
+import { v4 as uuidv4 } from 'uuid';
 
 const kMainDBName = 'appDB';
-export const kMainImageName = 'mainImage';
+
+// kCurrentBlobIDKey is the key into the miscStringVals store for the
+// FileUniqueID of the "current" blob (ie, the one that will be displayed
+// when the app is first opened).
+const kCurrentBlobIDKey = 'currentBlobID';
```

Comment:
Will we remove this again when we have proper multi-file support?

Reply:
Quite possibly, but I think that depends on the UX we land on.

Off the top of my head, the two obvious directions we could go are:

1. launch into a "gallery" or image picker sort of UI, where the user picks which image to show (possibly with a special case to skip straight to the image if only 1 exists), or
2. launch into a single (or a small amount of?) full image[s] based on either which one[s] were displayed most recently or which one[s] the user has chosen to "pin"

If we go with some variation of the latter option, then we'll still need to have some concept of the "current" or "pinned" image[s] to display as soon as the app is launched